1.7/administration/installing/custom/cli.md

```diff
@@ -304,19 +304,27 @@ To install DC/OS:
 
     ![alt text](../img/ui-installer-login.gif)
 
-8. Archive your installer files for safe-keeping. You'll need this archive to make new agents, including the [public agent][11].
+    You are done!
```

Comment:
Would like to see the backup step as a part of the canonical process (i.e. before "done")